src/main/scala/se/nimsa/sbx/box/BoxPushActor.scala

```diff
@@ -16,189 +16,127 @@
 
```

Comment:
`BoxPushActor` now contains very little functionality. It is responsible for polling for new outgoing transactions to send, and to cancel (remove) transactions. Functionality for transferring images is added through the `BoxPushOps` trait.